sdk/schemaregistry/schema-registry-avro/CHANGELOG.md

```diff
@@ -6,6 +6,8 @@
 
 ### Breaking Changes
 
+- `schemaGroup` is no longer a constructor parameter and has been moved to the constructor options.
```

Comment:
Might be worth noting that this is still (and only) required for serialization.

Reply:
Done.